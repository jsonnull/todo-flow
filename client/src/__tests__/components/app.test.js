// @flow

import React from 'react';
import ReactDOM from 'react-dom';
import { Provider } from 'react-redux';
import configureMockStore from 'redux-mock-store';

import App from '../../components/app';

const middlewares = [];
const mockStore = configureMockStore(middlewares);

describe('App', () => {
  it('renders without crashing', () => {
    const div = document.createElement('div');
    ReactDOM.render(
      <Provider store={mockStore({ todos: [], visibilityFilter: 'SHOW_ALL' })}>
        <App />
      </Provider>,
      div
    );
  });
});
