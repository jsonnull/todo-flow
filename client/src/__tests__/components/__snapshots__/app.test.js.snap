exports[`App renders without crashing 1`] = `
<div>
  <div>
    <form
      onSubmit={[Function]}>
      <input />
      <button
        type="submit">
        Add Todo
      </button>
    </form>
  </div>
  <ul />
  <p>
    Show:
     
    <span>
      All
    </span>
    , 
    <a
      href="#"
      onClick={[Function]}>
      Active
    </a>
    , 
    <a
      href="#"
      onClick={[Function]}>
      Completed
    </a>
  </p>
</div>
`;
